### PR TITLE
Add custom style for small devices' screen

### PR DESCRIPTION
**Pull request by edykim**

### CODE REVIEW

**Comment by Pluu** [comment]
thank you @edykim 
I've a question because there is no PR content.
Is it about fixing the width of a small device to 787px?

File: assets/css/custom.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@media (max-width: 768px) {
```
Comment (Pluu) [inline assets/css/custom.css]: @edykim 
The value in Bootstrap CSS is 767px.
Isn't there any problem unlike Bootstrap CSS?
https://github.com/Pluu/pluu.github.io/blob/master/assets/css/bootstrap.css#L1585

Comment (edykim) [reply assets/css/custom.css]: Thanks for pointing out, I updated the number as you suggested.

**Comment by edykim** [comment]
I updated the number as you suggested. Now it matched as a bootstrap medium width layout.

**Comment by Pluu** [comment]
LGTM!! 
image

**Pull request merged**